ci/infra/testrunner/testrunner.py

```diff
@@ -39,15 +39,18 @@ def build_skuba(options):
 
 def bootstrap(options):
     skuba = Skuba(options.conf, options.platform)
-
-    skuba.cluster_init()
+    skuba.cluster_init(kubernetes_version=options.kubernetes_version)
     skuba.node_bootstrap()
 
 
 def cluster_status(options):
     print(Skuba(options.conf, options.platform).cluster_status())
 
 
+def cluster_upgrade_plan(options):
+    print(Skuba(options.conf, options.platform).cluster_upgrade_plan())
```

Comment:
is this print going to show during tests?

Reply:
hmm good question, i also don't know for what we have the `print` wrapper on some method calls. maybe @pablochacin can shed some light here

Reply:
It shouldn't be there. Print is only for the commands that are expected to return an output to the user, like `status` and and `info`